integrations/torchvision/tutorials/sparse-transfer-learning.md

```diff
@@ -0,0 +1,168 @@
+# Sparsifying Image Classification Models with the CLI
+
+This page explains how to fine-tune a pre-sparsified Torchvision image classification models like ResNet-50 with SparseML's CLI.
+
+## Overview
+
+Sparse transfer learning is quite similiar to the typical transfer learning training, where we fine-tune a checkpoint pretrained on ImageNet onto a smaller downstream dataset. However, with Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity while the training process occurs.
```

Comment:
We need to decide on the capitalization for Sparse Transfer Learning / Sparse transfer learning for consistency

Reply:
similiar -> similar